tests/test_Step.cc

```diff
@@ -632,10 +632,13 @@ TEST_CASE("execute(): Exporting variables into a context", "[Step]")
 
     SECTION("No exported variables")
     {
+        // Variable 'b' is not touched (imported into any Step), so if it holds
+        // a char const* it will not be converted to std::string.
         context.variables["b"] = "Test";
         step.execute(context);
         REQUIRE(context.variables.size() == 1);
-        REQUIRE(std::get<std::string>(context.variables["b"]) == "Test");
+        // REQUIRE(std::get<std::string>(context.variables["b"]) == "Test"s);
+        REQUIRE(std::get<char const*>(context.variables["b"]) == "Test"s);
```

Comment:
This is where the solution with `std::string` vs. `const char*` gets weird. In a pinch, I could live with that, but let's consider if there are other alternatives first...

Reply:
If it gets wired I would avoid it. Periode. Again: why do we wanted to add `char const*` to the `std::variant`?